dts/bindings/sensor/honeywell,mpr.yaml

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+
+description: Honeywell Digital Pressure Sensor
+
```

Comment:
Is there a datasheet URL or something you can possible reference here?

Reply:
Ok, added.